Patch on YML field documentation (#203)
fivetran/dbt_quickbooks: CHANGELOG.md

# dbt_quickbooks v1.5.1
[PR #203](https://github.com/fivetran/dbt_quickbooks/pull/203) includes the following updates:

## Documentation

> - Corrects field documentation to match what's used in our models from `tax_rate` and `purchase_order` tables.

Reviewer: ```suggestion
- Corrects source and staging field documentation for the `tax_rate` and `purchase_order` tables.
```
